font-codegen/src/lib.rs

```diff
@@ -30,7 +32,11 @@ pub fn generate_code(code_str: &str, mode: Mode) -> Result<String, syn::Error> {
     // Generation is done in phases (https://github.com/googlefonts/fontations/issues/71):
     // 1. Parse
     debug!("Parse (mode {:?})", mode);
-    let mut items: Items = syn::parse_str(code_str)?;
+    // This is the one step where we can't readily intercept the error with logged_syn_error
+    let mut items: Items = syn::parse_str(code_str).map_err(|e| {
+        debug!("{}", Backtrace::capture());
```

Comment:
Not sure if this is intentional or not, but this will only capture a bactrace if the `RUST_BACKTRACE` env var is set. If we want to *always* capture a backtrace, we can use [`Backtrace::force_capture`](https://doc.rust-lang.org/1.65.0/src/std/backtrace.rs.html#311).

Reply:
Not capturing when backtrace is off feels correct to me, do you think we should always capture?